Cargo.lock

```diff
@@ -119,7 +119,7 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
was the update of the packages here intentional?

Reply:
Yeah, a leftover, deleted.